README.md

```diff
@@ -30,7 +30,12 @@ There are two main categories of samples: **Camera** and **Applications**. The s
       - [**ReadIterateZDF**][ReadIterateZDF-url] - Read point cloud data from a ZDF file, iterate through it, and extract individual points.
       - [**ZDF2PLY**][ZDF2PLY-url] - Convert point cloud from ZDF a file to a PLY file.
   - **Advanced**
-    - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration
+  - [**HandEyeCalibration**][HandEyeCalibration-url]
+      - [**HandEyeCalibration**][HandEyeCalibrationSample-url] - Perform Hand-Eye calibration.
+      - [**UtilizeEyeInHandCalibration**][UtilizeEyeInHandCalibration-url] - Transform single data point or entire point cloud from camera frame to robot base frame using Eye-in-Hand calibration matrix.
+	  - **Dependencies:**
+		  - [Math.NET Numerics](https://numerics.mathdotnet.com/#Math-NET-Numerics) version 4.12.0 or newer
+          - [Yaml.NET](https://github.com/aaubry/YamlDotNet/wiki) version 8.1.2 or newer
```

Comment:
There are no instructions on how to install/download or whatever needs to be done using these dependencies.

I suppose we should include instructions on: https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/534708242/Build+C+Samples+using+Visual+Studio

In addition, the readme file should have a link to the above article. Same as we have for C++ see:
![image](https://user-images.githubusercontent.com/46567886/91553916-7c377c80-e92e-11ea-95df-7687135c6900.png)

Reply:
Updated readme to be more similar like the C++ readme

Reply:
Should we add this link to our instructions? https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-powershell
I didin't know what package manager console is.

Reply:
It is something that is inside Visual Studio, but I can explain how to open it and if they dont have it, install it using this link. (Better to be safe the sorry)

Reply:
I don't think we need any instructions for this, since it happens automatically during build. What do you think?